src/app/components/tree-view/index.html

```diff
@@ -103,18 +107,14 @@ <h3>States</h3>
     <sky-demo-page-content sectionHeading="Related information">
       <h3>Components</h3>
       <ul>
-        <li><a routerLink="/components/flyout">Split view</a></li>
+        <li><a routerLink="/components/split-view">Split view</a></li>
```

Comment:
Should we include the list component in the related components? I mean, if the hierarchical view isn't necessary, that's the first alternative, isn't it?

Reply:
probably should link to Grid and Repeater as generic views.

Reply:
Sounds good. I can add links to grid and repeater. But what about list?

Reply:
not sure, since tree view doesn't work with List as far as I know

Reply:
I added links to grid and repeater, but not to list.

Reply:
Links seem to work, so resolving this issue.